### PR TITLE
Fix example documentation for selector-pseudo-element-colon-notation.

### LINKED ISSUE #927: Fix primary options for selector-pseudo-element-colon-notation in example config
Example config currently has ```"selector-pseudo-element-colon-notation": true,```

When the possible options for this rule are ``"single"`` and ``"double"``.

I have this corrected, and plan on doing a PR.

### PR DESCRIPTION
**Pull request by useffc**
Resolves #927.

### CODE REVIEW

**Comment by ntwb** [comment]
Thanks, looks good to me, LGTM :+1:

File: docs/user-guide/example-config.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ You might want to learn a little about [the conventions](https://github.com/styl
     "selector-no-type": true,
     "selector-no-universal": true,
     "selector-no-vendor-prefix": true,
-    "selector-pseudo-element-colon-notation": true,
+    "selector-pseudo-element-colon-notation": "single"|"double",
     "selector-root-no-composition": true,
     "string-no-newline": true,
     "string-quotes": true,
```
Comment (davidtheclark) [inline docs/user-guide/example-config.md]: I just noticed that this rule needs the same treatment.

```
"string-quotes": "single"|"double"
```

Would you mind adding that to the PR?

Comment (useffc) [inline docs/user-guide/example-config.md]: Good catch. Added.

**Comment by davidtheclark** [comment]
Thanks!

**Pull request merged**